husky_components/datagrid/decorators/tiles-view.js

```diff
@@ -166,16 +166,23 @@ define(function() {
         renderRecord: function(record) {
             record = processContentFilters.call(this, record);
             var title = this.getTitleString(record),
-                description = this.getDescriptionSring(record);
+                description = this.getDescriptionSring(record),
+                lineHeight, countLinesOfTitle;
+
             this.$tiles[record.id] = $(_.template(templates.tile, {
                 icon: this.options.icon,
                 title: title,
-                croppedTitle: this.sandbox.util.cropMiddle(this.getTitleString(record), 20),
-                description: this.sandbox.util.cropMiddle(description, 20)
+                description: this.sandbox.util.cropMiddle(description, 18)
             }));
             this.$tiles[record.id].data('id', record.id);
 
             this.$el.append(this.$tiles[record.id]);
+
+            lineHeight = parseInt(this.$tiles[record.id].find('.title').css('line-height').replace('px', ''));
+            countLinesOfTitle = this.$tiles[record.id].find('.title').height() / lineHeight;
+            if (countLinesOfTitle > 2) {
+                this.$tiles[record.id].find('.title').html(this.sandbox.util.cropMiddle(title, 18));
```

Comment:
Only the description is cropped above. What's happening here is the following. The whole title (without cropping) is rendered into the item. In this `if` you commented on, it gets checked if the title fits into two lines, if so everything is ok. However if the title would need more than two lines, it gets cropped to fit into a single row. This is the desired behavior discussed with @berndhep.